src/pages/billing/NewAccountModal.js

```diff
@@ -0,0 +1,339 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
+import { div, h, label } from 'react-hyperscript-helpers'
+import { buttonPrimary, buttonSecondary, Checkbox, Clickable, Select } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { validatedInput } from 'src/components/input'
+import Modal from 'src/components/Modal'
+import { authStore } from 'src/libs/auth'
+import colors from 'src/libs/colors'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import validate from 'validate.js'
+
+
+export default ({ onDismiss }) => {
```

Comment:
nit: It's nice to make a `const` and then export it, so that the React devtools can give it a meaningful name.

Reply:
IntelliJ's find usages is also easier to use if there's an explicit name. Otherwise, you have to know to invoke find usages on the `default` keyword (which I think I discovered by accident).